pgmpy/independencies/Independencies.py

```diff
@@ -195,7 +195,14 @@ def __str__(self):
     __repr__ = __str__
 
     def __eq__(self, other):
-        return self.get_assertion() == other.get_assertion()
+        if not isinstance(other, IndependenceAssertion):
+            return False
```

Comment:
@khalibartan Thank you. I guess i'll have to add test cases for the ```__eq__``` method too.